spec/models/reservation_spec.rb

```diff
@@ -21,6 +21,43 @@
   # can't run if nil (?)
   #
 
+  describe '.number_for' do
+    before(:each) do
+      @source = FactoryGirl.build_pair(:valid_reservation)
```

Comment:
FYI (and this should likely be dealt with in #1412) - even though we're calling `build` here, this still hits the database as the reservation factory is tied to the equipment model factory and creates new equipment models for *each* reservation it builds if they're not specified. Just something to keep in mind, we should probably [slim down our factories](https://robots.thoughtbot.com/factories-should-be-the-bare-minimum), but let's leave this for now.